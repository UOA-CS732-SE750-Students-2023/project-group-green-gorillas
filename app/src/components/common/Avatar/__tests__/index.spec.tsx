import React from "react";
import { render } from "@testing-library/react";
import { Avatar } from "../index";

describe("Avatar", () => {
  it("should be render correctly", () => {
    const { container } = render(<Avatar text={"Oliver Deng"} />);

    expect(container).toMatchSnapshot();
  });
});
