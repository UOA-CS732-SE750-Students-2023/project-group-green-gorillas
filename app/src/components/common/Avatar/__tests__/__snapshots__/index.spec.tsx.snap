// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should be render correctly 1`] = `
<div>
  <div
    class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-orach2-MuiAvatar-root"
  >
    OD
  </div>
</div>
`;
